### PR TITLE
Helm action to open file

### PR DESCRIPTION
**Pull request by Titan-C**
I wanted to find and edit the files of some of my contacts. I added a new Helm action that lets me jump to the  vcf file and edit it manually.

### CODE REVIEW

File: vdirel.el [hunk 1]
```diff
@@ -204,6 +205,11 @@ CANDIDATE is ignored."
                      (helm-marked-candidates)
                      ", ")))
 
+(defun vdirel--open-file (candidate)
+  "Open file assosiated to CANDIDATE."
```
Comment (DamienCassou) [inline vdirel.el]: can you please rephrase this docstring so it clearly says that `CANDIDATE` is ignored? You can copy/paste from `vdirel--helm-insert-contact-email`.

**Review by DamienCassou — CHANGES_REQUESTED** [review]
Excellent job, thank you. Can you please just take care of my comment on the docstring?

**Comment by DamienCassou** [comment]
Did you know about [khardel](https://github.com/DamienCassou/khardel)? I use it instead of vdirel.

**Comment by DamienCassou** [comment]
khardel has builtin support for contact edition.

**Comment by Titan-C** [comment]
I had no idea of khardel. It never showed up in any of my searches. It
does seem a lot more practical, only with the extra inconvenience of
installing an extra dependency.

**Review by DamienCassou — APPROVED** [review]
Excellent job! Thank you!

**Comment by DamienCassou** [comment]
You won't regret installing `khard`, it's a great tool.

**Pull request merged**